src/style.css

```diff
@@ -1,3 +1,72 @@
+:root {
+  --main-color-font: #000;
+  --main-color-font-hover: #00acdc;
+}
+
+* ,
+*::after,
+*::before {
+  margin: 0;
+}
+
 body {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+}
+
+header {
+  display: flex;
+  padding: 0 50px;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav__list {
+  display: flex;
   margin: 0;
+  padding: 0;
+  list-style-type: none;
+}
+
+.nav__item {
+  list-style: none;
+}
+
+.nav__item:not(:last-child) {
+  margin-right: 20px;
+}
+
+.nav__link {
+  color: var(--main-color-font);
+  text-decoration: none;
+  padding: 23px 0;
+  text-transform: uppercase;
+  font-size: 12px;
+  display: block;
+}
+
+.is-active,
+.nav__link:hover {
+  color: var(--main-color-font-hover);
+}
+
+.is-active {
+  position: relative;
+}
+
+.is-active:after {
```

Comment:
to access pseudoelements we use `::` operator, `:` is for pseudoclasses 😉 

Other than that, all good 🚀

Reply:
Fixed. New changes pushed.